python/README.md

```diff
@@ -124,7 +127,7 @@ pre-commit
 
   * If we already know some of the libraries we want to include, we can simply append those to the end:
     ```
-    conda create -n myenv python=3.9 pip black flake8 pre-commit pytest mypy docformatter
+    conda create -n myenv python=3.9 pip black flake8 pre-commit pytest mypy docformatter pydocstyle=6.1.1
```

Comment:
Is there a reason for requiring this specific version and not this as a minimum version, i.e., `pydocstyle>=6.1.1` ? (Also below)

Reply:
The version was set to match the one in the pre-commit. I changed to minimum version.